engine/rpcserver.go

```diff
@@ -4665,7 +4660,7 @@ func (s *RPCServer) GetFundingRates(ctx context.Context, r *gctrpc.GetFundingRat
 	if !a.IsFutures() {
 		return nil, fmt.Errorf("%s %w", a, order.ErrNotFuturesAsset)
 	}
-	start := time.Now().AddDate(-1, 0, 0)
+	start := time.Now().AddDate(0, -10, 0)
```

Comment:
ME: ?

Reply:
-10 months seems strange and accidental. Will change to -1 month